src/platforms/android/common/enriching-events/attachments.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Attachments Preview"
+sidebar_order: 106
+
+description: "Learn more about attachments, which provide the ability to store additional files to an event."
+---
+
+__Attention: This is a preview API, available since 4.0.0-alpha.2. This API is not in stable state yet. It may be renamed, changed or even removed in a future version.__
+
+Attachments provide the ability to to store additional files to an event. Be aware that attachments don't work yet with crashes.
+
+
+## Uploading Attachments
+
+You can either pass bytes or use a pathname to a file to create an attachment.
+
+```java
```

Comment:
We're trying to double down on Kotlin examples going forward.
Would you add it after the Java ones please?

Once we have all examples for both, we could swap them so Kotlin comes first for Android, what do you think?

Reply:
I agree fully, good idea 😀. I added Kotlin samples.